assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -435,6 +441,39 @@ export const DEFAULT_NOTIFICATIONS = {
 		},
 		isDismissible: true,
 	},
+	'first-party-mode-setup-cta-banner': {
+		Component: FirstPartyModeSetupBanner,
+		priority: 320,
+		areaSlug: NOTIFICATION_AREAS.BANNERS_BELOW_NAV,
+		groupID: NOTIFICATION_GROUPS.SETUP_CTAS,
+		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+		checkRequirements: async ( { select, resolveSelect } ) => {
+			if ( ! isFeatureEnabled( 'firstPartyMode' ) ) {
+				return false;
+			}
+
+			const analyticsModuleConnected =
+				select( CORE_MODULES ).isModuleConnected( 'analytics' );
+			const adsModuleConnected =
+				select( CORE_MODULES ).isModuleConnected( 'ads' );
+
+			const FPMSettings = await resolveSelect(
+				CORE_SITE
+			).getFirstPartyModeSettings();
+
+			const isFirstPartyModeEnabled = FPMSettings?.isEnabled;
+			const isFPMHealthy = FPMSettings?.isFPMHealthy;
+			const isScriptAccessEnabled = FPMSettings?.isScriptAccessEnabled;
+
+			return (
+				( analyticsModuleConnected || adsModuleConnected ) &&
+				! isFirstPartyModeEnabled &&
+				isFPMHealthy &&
+				isScriptAccessEnabled
+			);
```

Comment:
Let's return early if neither module is connected, to potentially avoid an unwanted request to retrieve the FPM settings.

The `FPMSettings` constant should start with a lowercase letter - however I'd actually suggest a refactor to use the individual selectors for each of the settings. I'll apply this change myself to save the round trip:

```suggestion
			const { isModuleConnected } = select( CORE_MODULES );

			if (
				! (
					isModuleConnected( 'analytics' ) ||
					isModuleConnected( 'ads' )
				)
			) {
				return false;
			}

			await resolveSelect( CORE_SITE ).getFirstPartyModeSettings();

			const {
				isFirstPartyModeEnabled,
				isFPMHealthy,
				isScriptAccessEnabled,
			} = select( CORE_SITE );

			return (
				! isFirstPartyModeEnabled() &&
				isFPMHealthy() &&
				isScriptAccessEnabled()
			);
```